google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -35,6 +33,7 @@
 import java.util.logging.LogRecord;
 import java.util.logging.Logger;
 import java.util.logging.SimpleFormatter;
+import com.google.cloud.logging.Logging.WriteOption;
```

Comment:
Import isn't sorted, trying running `git diff -U0 master | google-java-format-diff.py -p1 -i`? The python script is [here](https://github.com/google/google-java-format/tree/master/scripts).